### ISSUE TITLE
Decompile

### ISSUE THREAD

**Issue by UltimateBoi** [nexe/nexe#964]
Is there any way to decompile a application compiled by nexe?

**Comment by calebboyd** [comment]
Yes, applications can be 'decompiled'. The application code is not obfuscated at all. You can think of it like a node binary glued to a zip file.

**Comment by UltimateBoi** [comment]
or is it just a zip?
